users/vieting/experiments/switchboard/hybrid/feat/experiments.py

```diff
@@ -213,10 +213,27 @@ def run_specaug_scf():
 
     nn_args = get_nn_args_baseline(
         nn_base_args={
-            "scf": dict(
-                returnn_args=dict(batch_size=7000, specaug_mask_sorting=True),
+            "scf": dict(returnn_args=dict(batch_size=7000, specaug_mask_sorting=True),
                 feature_args={"class": "ScfNetwork", "size_tf": 256 // 2, "stride_tf": 10 // 2}
-            )
+            ),
+            "scf_divisor-4": dict(
+                returnn_args=dict(
+                    batch_size=7000, 
+                    specaug_mask_sorting=True,
+                    mask_divisor=2,
```

Comment:
This does not seem to be what the experiment name suggests.

Reply:
Fixed